src/main/scala/is/hail/driver/Count.scala

```diff
@@ -32,7 +29,7 @@ object Count extends Command {
 
     val (nVariants, nCalledOption) = if (options.genotypes) {
       val (nVar, nCalled) = vds.rdd.map { case (v, (va, gs)) =>
-        (1L, gs.count(_.isCalled).toLong)
+        (1L, gs.hardCallIterator.count(_ >= 0).toLong)
```

Comment:
I clearly don't understand what you're asking for. hardCallIterator creates an HardCallGenotypeStreamIterator which is an IntIterator which is unboxed.